### PR TITLE
Port scalable UI's to updated mark-2 designs

### PR DESCRIPTION
**Pull request by AIIX**
#### Description

- Scalable UI's ported to use the mark-2 design for currently existing scalable pages 
- Fixes crash caused by previous scalable UI's

#### Type of PR
- [ ] Bugfix
- [ ] Feature implementation
- [x] Refactor of code (without functional changes)
- [ ] Documentation improvements
- [ ] Test improvements

#### Testing
Test "what is the current weather" on Mycroft-GUI Application on the Desktop

#### CLA
- [x] Yes

### CODE REVIEW

File: ui/DailyColumnScalable.qml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+// limitations under the License.
+
+/*
+Define the default values for a weather forecast column.
+
+A forecast screen has four columns, each with the same data.  This abstracts out the
+common bits of each column.
+
+This code is specific to the Mark II device.  It uses a grid of 16x16 pixel
+squares for alignment of items.
```
Comment (krisgesling) [inline ui/DailyColumnScalable.qml]: Copy pasted across - can be deleted.

File: ui/current_1_scalable.qml [hunk 2]
```diff
@@ -23,46 +23,163 @@ device not conforming to the Mark II screen's form factor.
...
+                    //Current temperature in the configured temperature unit
+                    id: temperature
+                    font.weight: Font.Bold
+                    font.pixelSize: parent.height > parent.width ? parent.width * 0.65 : parent.height * 0.65
+                    anchors.baseline: parent.bottom
+                    anchors.baselineOffset: -Mycroft.Units.gridUnit
+                    horizontalAlignment: Text.AlignHCenter
+                    width: parent.width
+                    rightPadding: -font.pixelSize * 0.1
+                    font.family: "Noto Sans Display"
```
Comment (krisgesling) [inline ui/current_1_scalable.qml]: This is probably left in from before, but we've been trying to move away from setting the font-family too much to allow projects to use their own preferred fonts. 

If we need it to ensure sizing etc, we've been moving to "Noto Sans" without the "Display"

**Review by krisgesling — COMMENTED** [review]
This is great, thanks Aditya!

Looking good on desktop and no impact on the Mark II as expected.

I think it's a good move to leave the animations out for the moment, our priority is for it to be solid rather than fancy right now.

Two tiny tidy up things below:

File: ui/DailyColumnScalable.qml (hunk 1 above)
Comment (AIIX) [reply ui/DailyColumnScalable.qml]: Has been fixed

File: ui/current_1_scalable.qml (hunk 2 above)
Comment (AIIX) [reply ui/current_1_scalable.qml]: Has been fixed, I have set the font.family to general "Noto Sans"

**Comment by krisgesling** [comment]
Great, thanks!

**Pull request merged**